modules/ROOT/pages/multi-tenancy-and-tenant-configuration.adoc

```diff
@@ -1,297 +1,16 @@
 
 [WARNING]
 ====
 
-*Important* +
-Once a tenant has been created and activated, default theme is not yet loaded. +
-Therefore, as long as you have not logged in to the new tenant, the login page will miss the default theme.
-
-After the first login, default theme will be loaded and the login page will look as usual.
-====
-
-=== REST API
-
-The xref:platform-api.adoc[platform REST API] is a REST layer around the Java PlatformAPI to create the tenant.
-
-==== Walk-through
-
-===== Install `curl` command line tool
-
-`curl` is available on Linux OS and it transfers data from or to a server with various protocols such as HTTP and HTTPS.
-
-[source,bash]
-----
-$ sudo apt install curl
-----
-
-NOTE: this is to be done only once.
-
-===== Start a tomcat
-
-* Download a BonitaSubscription-7.x.y.zip
-* Unzip it
-* Provide a valid license file
-* Start the tomcat
-
-===== Login
-
-[source,bash]
-----
-$ curl -v -c saved_cookies.txt -X POST --url 'http://localhost:8080/bonita/platformloginservice' \
---header 'Content-Type: application/x-www-form-urlencoded; charset=utf-8' \
--d 'username=platformAdmin&password=platform' -O /dev/null
-----
-
-The response to this REST API call (HTTP) generates 2 cookies, which must be transfered with each subsequent calls.
-One of the cookie is `X-Bonita-API-Token`.
-
-Note that the security against CSRF attacks is enabled by default for all fresh installations; the subsequence REST API calls using DELETE, POST, or PUT HTTP methods must define the `X-Bonita-API-Token` header, with the value transmitted via the associated cookie.
-
-The cookies have been saved on the disk, in the `saved_cookies.txt` file:
-
-[source,bash]
-----
-$ cat saved_cookies.txt
-
-#HttpOnly_localhost	FALSE	/bonita/	FALSE	0	JSESSIONID	46EF8A05819B6C268EE700F3C3FC939A
-localhost	FALSE	/	FALSE	0	X-Bonita-API-Token	a94cbf84-6b71-409a-981f-f91b17466929
-----
-
-===== Create the new tenant
-
-[source,bash]
-----
-$ curl -b saved_cookies.txt -X POST 'http://localhost:8080/bonita/API/platform/tenant' \
--H "Content-Type: application/json" \
--H 'X-Bonita-API-Token: a94cbf84-6b71-409a-981f-f91b17466929' \
--d '{"name":"MyTenant", "description":"My tenant", "username":"install", "password":"install"}'
-
-{
-  "id": "101",
-  "creation": "2017-06-09 15:11:01.191",
-  "icon": "/default.png",
-  "username": "",
-  "description": "My tenant",
-  "name": "MyTenant",
-  "state": "DEACTIVATED",
-  "password": ""
-}
-----
-
-The new tenant has the id `101` and its state is `DEACTIVATED`
-
-===== Activate the tenant with id `101`
-
-[source,bash]
-----
-$ curl -v -b saved_cookies.txt -X PUT 'http://localhost:8080/bonita/API/platform/tenant/101' \
--H "Content-Type: application/json" \
--H 'X-Bonita-API-Token: a94cbf84-6b71-409a-981f-f91b17466929' \
--d '{"state":"ACTIVATED"}'
+Starting with Bonita 2021.1, this feature is *deprecated*, and as such, not offered to new customers nor as a Subscription modification for existing customers. + 
```

Comment:
⚠️ there is no mention to such deprecation in the release notes whereas the 2021.1 version has been released a few months ago.
It should at least mention it afterwards and could have a link to this new content for alternatives.

Reply:
good catch ! 
Nathalie, please add this mention in the RN of 2021.1

Reply:
👍